### DISCUSSION TITLE
Hyprland is very unstable in nvidia graphics!

### DISCUSSION THREAD

**Discussion by nicolaseliasx** [hyprwm/Hyprland#8445] [General]
Today, I decided to remove Hyprland from my computer. After each update, the system on NVIDIA cards seems to become increasingly unstable, with flickers happening constantly, which makes the experience of using Hyprland — one of the most interesting window managers — quite unpleasant. I’m planning to switch to a more stable alternative today. I used other Wayland compositors for a short time and didn’t experience these issues; apparently, it only happens here on Hyprland, which is unfortunate.

**Comment by izmyname** [comment]


**Reply by JJDizz1L** [reply]
Uhh, you know there's a Discord filled with many people who can help you through these issues =]

https://discord.gg/Tw7TCYXT

Come to the # hyprland-nvidia channel and depending on the distro and your hardware, we may can help you.

**Reply by vaxerski** [reply]
@izmyname please dont spam useless images or I will ban you

**Reply by izmyname** [reply]
@vaxerski no need for threats. Asking was just enough.

Anyway, let's move on.

**Comment by TeChn4K** [comment]
Not the case for me : after more than a year of intensive use of Hyprland, I'm a very happy user.
I've had very few crashes, and zero issue specifically linked to my graphics card. Generally, most of the issues I encountered came from system misconfiguration.

Workstation laptop with an NVIDIA GeForce RTX 3050 and 3 screens setup.
